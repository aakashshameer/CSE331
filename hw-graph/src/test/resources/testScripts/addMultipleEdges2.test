#Test to create three nodes, with multiple edges in the graph

#Create a graph
CreateGraph dgraph

#Add a node
AddNode dgraph a
AddNode dgraph b
AddNode dgraph c

#Add six edges
AddEdge dgraph a a AA
AddEdge dgraph a c AC
AddEdge dgraph b b BB
AddEdge dgraph b c BC
AddEdge dgraph b c BC2
AddEdge dgraph c a CA
AddEdge dgraph c b CB

#Print nodes in the graph
ListNode dgraph

#Print outgoing edges from a
ListChildren dgraph a

#Print outgoing edges from b
ListChildren dgraph b

#Print outgoing edges from c
ListChildren dgraph c