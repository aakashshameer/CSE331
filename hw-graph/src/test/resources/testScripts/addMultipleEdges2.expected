#Test to create three nodes, with multiple edges in the graph

#Create a graph
create graph dgraph

#Add a node
added node a to dgraph
added node b to dgraph
added node c to dgraph

#Add six edges
added edge AA from a to a in dgraph
added edge AC from a to c in dgraph
added edge BB from b to b in dgraph
added edge BC from b to c in dgraph
added edge BC2 from b to c in dgraph
added edge CA from c to a in dgraph
added edge CB from c to b in dgraph

#Print nodes in the graph
dgraph contains: a b c

#Print outgoing edges from a
the children of a in dgraph are: a(AA) c(AC)

#Print outgoing edges from b
the children of b in dgraph are: b(BB) c(BC) c(BC2)

#Print outgoing edges from c
the children of c in dgraph are: a(CA) b(CB)